pkg/apis/fission.io/v1/validation.go

```diff
@@ -304,6 +304,10 @@ func (spec FunctionSpec) Validate() error {
 		result = multierror.Append(result, spec.InvokeStrategy.Validate())
 	}
 
+	if spec.FunctionTimeout <= 0 {
+		result = multierror.Append(result, MakeValidationErr(ErrorInvalidValue, "FunctionTimeout value", spec.FunctionTimeout, "not a valid value. Should always be more than 0"))
```

Comment:
"FunctionTimeout value" -> "FunctionSpec.FunctionTimeout"

Consider the compatibility of old fission spec files which have no `FunctionTimeout` field may be broken by this change, please comment out the validation and prints warnings to users so that they can add miss field. You can take a look at following code snippets as an example.

1. Prints warning when a user uses spec command with old spec files.
https://github.com/fission/fission/blob/e481c234ee038f8ce8c877a4ab7d3fbf8b8b5597/pkg/fission-cli/cmd/spec/spec.go#L489-L492

2. I will let validate function return warning in later PR. Before that, let's comment out the validation part to prevent compatibility problem. 
https://github.com/fission/fission/blob/e481c234ee038f8ce8c877a4ab7d3fbf8b8b5597/pkg/apis/fission.io/v1/validation.go#L350-L353